x-pack/legacy/plugins/siem/public/components/ml_popover/upgrade_contents.tsx

```diff
@@ -38,14 +46,24 @@ export const UpgradeContents = React.memo(() => {
         />
       </EuiText>
       <EuiSpacer />
-      <EuiButton
-        href="https://www.elastic.co/subscriptions"
-        iconType="popout"
-        iconSide="right"
-        target="_blank"
-      >
-        {i18n.UPGRADE_BUTTON}
-      </EuiButton>
+      <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none">
+        <EuiButton
+          href="https://www.elastic.co/subscriptions"
+          iconType="popout"
+          iconSide="right"
+          target="_blank"
+        >
+          {i18n.UPGRADE_BUTTON}
+        </EuiButton>
+        <EuiButton
+          href={`${chrome.getBasePath()}/app/kibana#/management/elasticsearch/license_management`}
+          iconType="popout"
+          iconSide="right"
+          target="_blank"
+        >
```

Comment:
Since the management app is part of Kibana, I think it might be better to use `managementApp` for the `iconType` prop (as it's not an external link). Whether or not it should also have `target="_blank"` is debatable. I'm personally fine with it either way.

Also, I'm thinking we should remove the `iconSide` prop here (which will move the icon to the left side).